weejson-play/src/main/scala/com/rallyhealth/weejson/v1/play/PlayJson.scala

```diff
@@ -41,9 +41,9 @@ class PlayJson extends com.rallyhealth.weejson.v1.AstTransformer[JsValue] {
 
   def visitNull(): JsValue = JsNull
 
-  val visitFalse: JsValue = JsBoolean(false)
+  def visitFalse(): JsValue = JsBoolean(false)
```

Comment:
Yeah, I was going to do that and then thought "isn't that what mjson did that caused the memory leak?" and got spooked. Happy to make the change if you think it is safe. Let me know...

Reply:
Yes, please. `JsBoolean` is immutable, so nothing spooky here.

For reference: mjson objects can't be shared because of https://github.com/bolerio/mjson/blob/e7a4da2daa6e17a63ec057948bc30818e8f44686/src/java/mjson/Json.java#L1928-L1941

Reply:
done